client/my-sites/checkout/get-thank-you-page-url/test/get-thank-you-page-url.ts

```diff
@@ -1014,65 +995,37 @@ describe( 'getThankYouPageUrl', () => {
 		expect( url ).toBe( `/checkout/thank-you/foo.bar/${ samplePurchaseId }` );
 	} );
 
-	it( 'redirects to business upgrade nudge if jetpack is not in the cart, and premium is in the cart', () => {
-		const cart = {
-			...getMockCart(),
-			products: [
-				{
-					...getEmptyResponseCartProduct(),
-					product_slug: 'value_bundle',
-					bill_period: '365',
-				},
-			],
-		};
-		const url = getThankYouPageUrl( {
-			...defaultArgs,
-			siteSlug: 'foo.bar',
-			cart,
-			receiptId: samplePurchaseId,
-		} );
-		expect( url ).toBe( `/checkout/foo.bar/offer-plan-upgrade/business/${ samplePurchaseId }` );
-	} );
-
-	it( 'redirects to business monthly upgrade nudge if jetpack is not in the cart, and premium monthly is in the cart', () => {
-		const cart = {
-			...getMockCart(),
-			products: [
-				{
-					...getEmptyResponseCartProduct(),
-					product_slug: 'value_bundle_monthly',
-					bill_period: '31',
-				},
-			],
-		};
-		const url = getThankYouPageUrl( {
-			...defaultArgs,
-			siteSlug: 'foo.bar',
-			cart,
-			receiptId: samplePurchaseId,
-		} );
-		expect( url ).toBe(
-			`/checkout/foo.bar/offer-plan-upgrade/business-monthly/${ samplePurchaseId }`
-		);
-	} );
-
-	it( 'redirects to the business upgrade nudge with a placeholder when jetpack is not in the cart and premium is in the cart but there is no receipt', () => {
-		const cart = {
-			...getMockCart(),
-			products: [
-				{
-					...getEmptyResponseCartProduct(),
-					product_slug: 'value_bundle',
-				},
-			],
-		};
-		const url = getThankYouPageUrl( {
-			...defaultArgs,
-			siteSlug: 'foo.bar',
-			cart,
-		} );
-		expect( url ).toBe( `/checkout/foo.bar/offer-plan-upgrade/business/:receiptId` );
-	} );
+	it.each( [
+		PLAN_PERSONAL_MONTHLY,
+		PLAN_PERSONAL_MONTHLY,
+		PLAN_PERSONAL,
+		PLAN_PERSONAL_2_YEARS,
+		PLAN_PERSONAL_3_YEARS,
+		PLAN_PREMIUM_MONTHLY,
+		PLAN_PREMIUM,
+		PLAN_PREMIUM_2_YEARS,
+		PLAN_PREMIUM_3_YEARS,
+	] )(
+		'does not redirect to business upgrade nudge if product_slug is %s (previously it was, and now we render standard thanks you page with upsell banner)',
```

Comment:
I'm not sure referring to paste state is useful (Git and GitHub will provide that information):
```suggestion
		'does not redirect to business upgrade nudge if product_slug is %s',
```